app/components/enter-new-account/component.js

```diff
@@ -72,12 +65,21 @@ export default class EnterNewAccountComponent extends Component {
         currency: this.currency
       });
       newRecord.save();
-      this.accountName = "";
-      this.interestRate = "";
-      this.currentValue = "";
-      this.accountType = "checking";
-      this.currency = "PLN";
+      this.clearAll()
     }
   }
 
+  isNotFilled(){
+    return isEmpty(this.accountName) ||
+      isEmpty(this.currentValue) ||
+      (isEmpty(this.interestRate) && isEmpty(this.isChecking))
```

Comment:
Consider some explanation variable at least for the last line with && - it should tell us what that means

Reply:
Done